internal/pkg/cli/pipeline_update.go

```diff
@@ -111,6 +112,7 @@ func (opts *UpdatePipelineOpts) getArtifactBuckets() ([]deploy.ArtifactBucket, e
 	return buckets, nil
 }
 
+// Execute destroys the old pipeline and creates a new one.
```

Comment:
does it actually delete the old pipeline? I think it "Creates a CodePipeline pipeline from the pipeline manifest if one does not already exist, otherwise it updates it."